h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -571,6 +571,124 @@ \subsubsection{Tweedie Models}
+
+\begin{equation} \label{eq:8}
+V_k = \frac{\Gamma(1+\alpha k)\phi^{k(\alpha -1)}(p-1)^{\alpha k}}{\Gamma(1+k)w^{k(\alpha -1)}(p-2)^ky^{\alpha k}}(-1)^k \sin (-k\pi \alpha)
+\end{equation}
+
+\waterExampleInR
+\lstinputlisting[style=R]{GLM_Vignette_code_examples/glm_tweedie_dispersion_example.R}
+
+\waterExampleInPython
+\lstinputlisting[style=python]{GLM_Vignette_code_examples/glm_tweedie_dispersion_example.py}
+
+\paragraph{Tweedie Variance Power Estimation} 
+
+Depending on $p$, $y$, and $\phi$, different methods are used for this log likelihood estimation. To start, let:
+
+$$\xi = \frac{\phi}{y^{2-p}}$$
+
+If $p=2$, then it will use the log likelihood of the Gamma distribution:
+
+$$\log (p) = \begin{cases} - \infty & y=0 \\ \frac{1}{\phi} \log (\frac{1}{\phi \mu}) - \log \text{Gamma} \frac{1}{\phi} + \log (y)(\frac{1}{\phi} -1) + (-\frac{1}{\phi \mu} y) & y>0 \\\end{cases}$$
+
+If $p=3$, then it will use the inverse Gaussian distribution:
+
+$$\log (p) = \begin{cases} - \infty & y=0 \\ \frac{1}{2} \Big(-log (\phi \mu) \log (2 \pi) -3 \log \big( \frac{y}{\mu} - \frac{(\frac{y}{\mu} -1)^2}{\phi \mu \frac{y}{\mu}} \Big) - \log (\mu) & y>0 \\\end{cases}$$
+
+If $p<2$ and $\xi \leq 0.01$, then it will use the Fourier inversion method.
+
+If $p>2$ and $\xi \geq 1$, then it will also use the Fourier inversion method.
+
+Everything else will use the Series method. However, if the Series method fails (output of \texttt{NaN}), then it will try the Fourier inversion method instead.
+
+If both the Series method and Fourier inversion method fail, or if the Fourier inversion method was chosen based on the $\xi$ criterium and it failed, it will then estimate the log likelihood using the Saddlepoint approximation.
+
+Here are the general usages for Tweedie variance power and dispersion parameter estimation using maximum likelihood:
+\begin{itemize}
+\item \texttt{fix\_tweedie\_variance\_power = True} and \\ \texttt{fix\_dispersion\_parameter = False} as it will use the Tweedie variance power set in parameter \texttt{tweedie\_variance\_power} and estimate the dispersion parameter starting with the values set in parameter \texttt{init\_dispersion\_parameter};
+\item  \texttt{fix\_tweedie\_variance\_power = False} and \\ \texttt{fix\_dispersion\_parameter = True} as it will use the dispersion parameter value set in parameter \texttt{init\_dispersion\_parameter} and estimate the Tweedie variance power starting with the value set in parameter \texttt{tweedie\_variance\_power};
+\item  \texttt{fix\_tweedie\_variance\_power = False} and \\ \texttt{fix\_dispersion\_parameter = False} as it will estimate both the variance power and dispersion parameter using the values set in \\ \texttt{tweedie\_variance\_power} and \texttt{init\_dispersion\_parameter} respectively.
+
```

Comment:
Minor change:

 \item  \texttt{fix\_tweedie\_variance\_power = False} and \\ \texttt{fix\_dispersion\_parameter = False} as it will estimate both the variance power and dispersion parameter using the values set in \\ \texttt{tweedie\_variance\_power} and \texttt{init\_dispersion\_parameter} respectively.

to

 \item  \texttt{fix\_tweedie\_variance\_power = False} and \\ \texttt{fix\_dispersion\_parameter = False} as it will estimate both the variance power and dispersion parameter staring with the values set in \\ \texttt{tweedie\_variance\_power} and \texttt{init\_dispersion\_parameter} respectively.